examples/kotlin/BUILD.bazel

```diff
@@ -21,28 +23,66 @@ kt_js_library(
     deps = [":kotlinx-html-js"],
 )
 
+# Copy boostrap to the bin folder as entry_point.js
```

Comment:
nit: boostrap

Reply:
bazel will complain about bootstrap.js matching the input... maybe `_bootstrap.js` for the bin folder.